src/ui/page/share/dataset/steps/summary/summary.scss

```diff
@@ -0,0 +1,101 @@
+@import '@/ui/style/fonts.scss';
+@import '@/ui/style/theme.scss';
+@import '@/ui/style/mixins';
+
+.okp4-dataverse-portal-share-dataset-summary-container {
+  @include with-theme {
+    @include grid-container($colGutter: 48px, $rowGutter: 22px);
+    grid-template-rows: auto 1fr 1fr;
+    height: 100%;
+    padding: 34px 10px 0 0;
+    box-sizing: border-box;
+    overflow-y: auto;
+
+    @include use-breakpoints(('desktop')) {
+      @include grid-container($colGutter: 34px, $rowGutter: 26px);
+    }
+
+    h2 {
+      @include grid-item($colStart: 1, $colEnd: -1, $rowStart: auto, $rowEnd: auto);
+      @include noto-sans(700);
+      font-size: 28px;
+      line-height: 28px;
+      letter-spacing: 0.4px;
+      color: themed('text');
+    }
+
+    .okp4-dataverse-portal-share-dataset-summary-left {
+      @include grid-item($colStart: 1, $colEnd: 6, $rowStart: auto, $rowEnd: auto);
+      box-sizing: border-box;
+      max-height: 690px;
+      background-color: themed('article-background');
+      border-radius: 12px;
+      padding: 15px 10px 15px 18px;
+
+      @include use-breakpoints(('desktop')) {
+        @include bi-column-item(left);
+        max-height: 520px;
+      }
+
+      .okp4-dataverse-portal-share-dataset-summary-text-container {
+        overflow-y: auto;
+        height: 100%;
+
+        .okp4-dataverse-portal-share-dataset-summary-text {
+          @include columns-with-gap(6px);
+
+          h3 {
+            @include noto-sans(700);
+            line-height: 24px;
+            text-transform: uppercase;
+            font-size: 11px;
+            color: themed('share-dataset-summary-title');
+          }
+
+          span {
+            @include noto-sans(400);
+            padding-bottom: 14px;
+            color: themed('text');
+            line-height: 22px;
+            font-size: 16px;
+            max-width: 500px;
+          }
+        }
+      }
+    }
+
+    .okp4-dataverse-portal-share-dataset-summary-right {
+      @include grid-item($colStart: 6, $colEnd: 13, $rowStart: auto, $rowEnd: auto);
+      @include columns-with-gap(136px);
+      box-sizing: border-box;
+      justify-content: space-between;
+
+      @include use-breakpoints(('desktop')) {
+        @include bi-column-item(right);
+        @include columns-with-gap(30px);
+      }
+
+      .okp4-dataverse-portal-share-dataset-summary-file-list {
+        overflow: auto;
+        margin: 0;
+        padding-right: 14px;
+        max-height: 430px;
+
+        @include use-breakpoints(('desktop')) {
+          max-height: 360px;
+        }
+      }
+    }
+
+    .okp4-dataverse-portal-share-dataset-summary-certify {
```

Comment:
This element should have more vertical margin between the summary and step nav bar